ui.frontend/src/main/components/WishlistPage/useAddToWishlistEvent.js

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ *
+ *    Copyright 2021 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+import { useEventListener } from '../utils/hooks';
+import defaultOperations from '@magento/peregrine/lib/talons/Wishlist/AddToListButton/addToListButton.gql';
+import mergeOperations from '@magento/peregrine/lib/util/shallowMerge';
+import { useMutation } from '@apollo/client';
+
+const useAddToWishlistEvent = (props = {}) => {
```

Comment:
Should we move this to the Core Components, similar to `useAddToCartEvent` ?

Reply:
The user can add to wishlist only if logged in.
So I call `useAddToWishlistEvent()` in the `accountTrigger` component.

Reply:
Calling it there is ok, but the function it self should be imported from the CIF Core Components (it is part of the glue between CIF and Peregrine, similar to`useAddToCartEvent`)

Reply:
sure, that can work